### DISCUSSION TITLE
5.3 Compatibility?

### DISCUSSION THREAD

**Discussion by tdhavema** [midgen/uesvon#29] [General]
Hey this is an insanely useful plugin. I am new to Unreal Engine, and I have some C++ learning but not much. I have found so many resources about SVO online and a few in C++, however I'm using 5.3 and I'm trying to accomplish the same thing. I tried looking through some of the file code to see if I can associate this with 5.3 and test as-is, but I can't seem to find that anywhere either. Are you planning a 5.3 update, or could you suggest resources of where to learn this in C++ for Unreal? I have found a few tutorials C++ for Vulkan API directly.

**Comment by midgen** [comment]
I have synced UE 5.3, will have a look this evening to see what needs fixing up. I'm pretty sure it was working OK on 5.2.

**Reply by tdhavema** [reply]
Thank you so much! I added to the .uplugin file the "EngineVersion" : "5.3" and it would not load at all. If I don't include that line and build from IDE it gives me the error of it was built using a different engine version.

**Reply by tdhavema** [reply]
The only reason I want to use this in 5.3 is simply because Epic also released a major update to GAS in 5.3 and I have been learning that as well. I would go back to 5.2 to use this, but I don't want to go backwards on GAS.

**Comment by midgen** [comment]
I've tested the plugin just now and it's working in 5.3 without any changes from the master branch.

I did come across this issue with building in the latest version of Visual Studio 2022 :

https://forums.unrealengine.com/t/error-compiling-the-automation-tool-after-updating-visual-studio-today-unreal-5-3-2/1393088/24

There's a workaround in there to get UE to build.

You will need to use a code project and build in Visual Studio to use the plugin, as the source needs compiling. 

I'll see if I can upload a binary for this version, but I won't make a habit of maintaining releases :)

**Comment by midgen** [comment]
Binaries here :

https://github.com/midgen/uesvon/releases/tag/v5.3
